doc/plutus-comparison.md

```diff
@@ -0,0 +1,179 @@
+# Comparing CTL and Plutus
+
+This document outlines the core differences between CTL and Plutus (particularly in the context of the Plutus Application Backend [PAB]). CTL is of course directly inspired by Plutus and PAB and we have attempted to preserve a high degree of similarity between the two APIs. In many cases, it should be possible to copy-paste existing code written for PAB deployments and adjust it for CTL fairly easily (accounting of course for existing differences between Haskell and Purescript). Nevertheless, CTL and Plutus differ in several important ways, as outlined below.
+
+Note that differences between Haskell and Purescript, while also relevant to such a comparison, is beyond the scope of this document unless such differences have a direct bearing on divergences between the two CTL and Plutus.
+
+**Table of Contents**
+
+- [Core conceptual differences](#core-conceptual-differences)
+  - [Library vs. process](#library-vs-process)
+  - [The `Contract` type](#the-contract-type)
+- [API differences](#api-differences)
+  - [Constraints and lookups](#constraints-and-lookups)
+  - [Typed scripts](#typed-scripts)
+  - [Working with scripts](#working-with-scripts)
+    - [Using scripts from the frontend](#using-scripts-from-the-frontend)
+    - [Applying arguments to parameterized scripts](#applying-arguments-to-parameterized-scripts)
+
+## Core conceptual differences
+
+### Library vs. process
+
+Unlike contracts written for PAB, which are compiled to a single process, CTL is a library. CTL itself can be [imported as a Purescript library](./using-ctl.md) and contracts written in CTL compile to Javascript that can be run in the browser or NodeJS. Accordingly, there is no need to activate endpoints in CTL -- contracts are executed by calling effectful functions written using the library. This distinction has influenced our adaption of Plutus' `Contract` type, as outlined [below](#the-contract-type).
+
+Note, however, that CTL still requires a number of runtime dependencies. In some respects, this is similar to PAB, which also needs to communicate with plutus-chain-index and a running node. Please see the [documentation](./runtime.md) for more details on CTL's runtime.
+
+### The `Contract` type
+
+Both CTL and Plutus define `Contract` monads for constructing, balancing, and submitting transactions. There are considerable differences between the two, however:
+
+**CTL**:
+
+```purescript
+newtype Contract (r :: Row Type) (a :: Type) = Contract (QueryMExtended r a)
+```
+
+where
+
+- `QueryMExtended` is an internal monad transformer stack based on `ReaderT` over `Aff` (Purescript's monad for asynchronous effects, with no Haskell analogue).
+- `r` extends the record serving as the reader environment
+
+**Plutus**:
+
+```haskell
+newtype Contract (w :: Type) (s :: Row Type) (e :: Type) (a :: Type) = Contract { ... }
```

Comment:
`Row *`

Reply:
I didn't reproduce the `Contract` definition verbatim. `*` is a soon-to-be-deprecated synonym for `Type` anyway